hypervisor/libvirt.go

```diff
@@ -43,6 +44,21 @@ func (d *LibvirtDriver) DomainNames() ([]string, error) {
 	return d.virshList()
 }
 
+// Version returns the version string for libvirt on the hypervisor.
+func (d *LibvirtDriver) Version() (string, error) {
+	out, err := virsh.Virsh(
+		d.prep,
+		d.uri.String(),
+		"version",
+	)
+	if err != nil {
+		return "", err
+	}
+
+	// Strip all newlines down to the version string
+	return strings.TrimSpace(string(out)), nil
```

Comment:
On my system, this reports:
```
[zsh|matt@nerr-2]:~ 0 % virsh version
Compiled against library: libvirt 1.2.2
Using library: libvirt 1.2.2
Using API: QEMU 1.2.2
Running hypervisor: QEMU 2.0.0
```

Shouldn't you need to process the output to get a version string?